src/styles/card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cardWidth;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $BorderColor;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: auto;
+  }
+
+  &__title {
+    margin-top: 40px;
+    margin-bottom: 0;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $blackColor;
+    overflow: hidden;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $grayColor;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+  }
+
+  &__priceInfo {
+    display: flex;
+    justify-content: space-between;
+    height: 18px;
+    margin-top: 24px;
+
+  }
```

Comment:
1. Not sure about `height`
<img width="1022" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/49764881/956e5c49-5c08-4358-b304-28cdef50f861">

2. Do not leave random empty lines

Reply:
You are right, we don't need height here also.